stellarphot/settings/tests/test_custom_widgets.py

```diff
@@ -267,6 +267,42 @@ def test_edit_button_returns_after_making_new_item(self, tmp_path):
         # The edit button should now be displayed
         assert choose_or_make_new._edit_button.layout.display != "none"
 
+    def test_save_button_disabled_when_no_changes(self, tmp_path):
+        # Immediately after the edit button has been clicked, the save button should be
+        # disabled because no changes have been made yet.
+        #
+        # That should remain true even after the user makes a change and then saves it
```

Comment:
This statement is unclear, I assume you mean this should remain true after a user makes a change and then saves it UNTIL they start editing and have made new changes or something like that.